apps/web/src/components/doc-editor/docSheet.css

```
@@ -65,10 +68,13 @@
   --doc-sheet-side-w: 287px;
   --doc-sheet-page-h: 1122px;
 
-  /* Document faces — independent of app chrome (`index.css` `--font-*`). */
+  /* Document faces — independent of app chrome (`index.css` `--font-*`).
+     Family / size / line-height are overridden from metadata.typography. */
   --doc-font-body: "IBM Plex Sans", "Helvetica Neue", Arial, sans-serif;
```

Stylelint (17.14.0)

[error] Expected "Arial" to be "arial" (value-keyword-case)

(value-keyword-case)